src/test/common/clara/test_testing_utils.cljc

```diff
@@ -0,0 +1,56 @@
+#?(:clj
```

Comment:
Having everything other than the name be separate is complicated by having multiple top-level forms beneath the declaration for clj and cljs.  I think I'll just keep this as is; by my count only the clara.rules require is identical between clj and cljs so splicing in lots of reader conditions would hurt readability more than it would help it in this instance IMO.